homeassistant/components/ihc/__init__.py

```diff
@@ -287,6 +300,11 @@ def set_runtime_value_float(call):
         value = call.data[ATTR_VALUE]
         ihc_controller.set_runtime_value_float(ihc_id, value)
 
+    async def pulse_runtime_input(call):
```

Comment:
We prefer to name functions that should be run in the event loop, callbacks or coroutines, starting with `async_`. Eg `async_pulse_runtime_input`.

Reply:
Fixed